exporter/signalfxexporter/metricdata_to_signalfxv2.go

```diff
@@ -174,14 +172,13 @@ func metricDataToSignalFxV2(
 						numDroppedTimeSeries++
 						logger.Warn(
 							"Timeseries for summary metric dropped",
-							zap.Error(err),
-							zap.String("metric", *sfxDataPoint.Metric))
```

Comment:
Same question here. I see you kept it below.

Reply:
The field is also in the error, feels like duplicate information for me.

Reply:
Added back, will cleanup later.